src/main/java/com/znsio/e2e/runner/Runner.java

```diff
@@ -239,6 +242,18 @@ public static String getWebDriverManagerProxyURL() {
     }
 
     public static String getBrowserConfigFileContents() {
+        InputStream inputStream;
+        String browserConfigFile = configs.get(BROWSER_CONFIG_FILE);
+        try {
+            if (browserConfigFile.contains("default")) {
+                inputStream = Runner.class.getResourceAsStream(DEFAULT_BROWSER_CONFIG_FILE);
+            } else {
+                inputStream = Files.newInputStream(Paths.get(browserConfigFile));
+            }
+        } catch(Exception e) {
+            throw new InvalidTestDataException("There was a problem while setting browser config file");
```

Comment:
we should also add which file(s) were tried for loading

Reply:
added file name in exception message